presto-phoenix/src/test/resources/hbase-site.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
```

Comment:
This file looks unused. If I am wrong then it would be nice to configure that from test code via some phoenix config object.